loopchain/configure_default.py

```diff
@@ -409,6 +409,18 @@ def is_support_node_function(cls, node_function, node_type):
 KMS_SECRET_KEY_LABEL = "KEY_ENCRYPTION"
 
 
+#########
+# HSM ###
+#########
+# HSM_ENABLE_USE has higher priority than use configuration of KMS.
+HSM_ENABLE_USE = False
+HSM_YUBI_CONNECTOR = "http://127.0.0.1:12345"
+HSM_YUBI_SESSION_ID = 1
+HSM_YUBI_SESSION_PASSWORD = "password"
```

Comment:
default password is `password`?

Reply:
Yes, the default password of the YubiHSM device.